js/index.js

```diff
@@ -0,0 +1,66 @@
+var today = new Date();
+var thisYear = today.getFullYear();
+const footer = document.querySelector("footer");
+const copyright = document.createElement("p");
+
+copyright.innerHTML = `<center> Musab ${thisYear} </center>`;
+footer.appendChild(copyright);
+
+const skills = ["Mathematic", "Phisic", "Chemstry", "Biology", "Astronomy", "Electric"];
+
+const skillsSection = document.getElementById("skills");
+
+const skillsList = skillsSection.lastElementChild;
+
+for (let i = 0; i <= skills.length; i++) {
+
+
+    let skill = document.createElement("li");
+
+    skill.innerHTML = skills[i];
+
+    skillsList.appendChild(skill);
+
+
+}
+
+let messageForm = document.getElementById("leave_message");
+
+messageForm.addEventListener("submit", function (event) {
+
+    event.preventDefault();
+    let name = event.target.name.value;
+    let email = event.target.email.value;
+    let textMessage = event.target.message.value;
+    console.log(`Entered name: ${name}`);
+    console.log(`Entered email: ${email}`);
+
+    let messageSection = document.getElementById("message");
+    let messageList = messageSection.querySelector("ul");
+
+    let newMessage = document.createElement("li");
+    newMessage.innerHTML = `<a href= mailto:${email}>${name}</a> <span> wrote : ${textMessage}  </span>`;
+
+    let removeButton = document.createElement("button");
+    removeButton.innerText = "remove";
+    removeButton.setAttribute("type", "button");
+    removeButton.addEventListener("click", (event) => {
+
+        let entry = removeButton.parentNode;
+        entry.parentNode.removeChild(entry);
```

Comment:
Here you could have also called `entry.remove();`. That would have also removed the node.